docs/sophia_features.md

```diff
@@ -628,10 +628,18 @@ arithmetic operations:
 - remainder (`x mod y`), satisfying `y * (x / y) + x mod y == x` for non-zero `y`
 - exponentiation (`x ^ y`)
 
-All operations are *safe* with respect to overflow and underflow. 
+All operations are *safe* with respect to overflow and underflow.
 The division and modulo operations throw an arithmetic error if the
 right-hand operand is zero.
 
+Sophia arbitrary-sized integers (FATE) also supports the following bitwise operations:
```

Comment:
I don't think you take endianness into account when shifting?! What would that even mean? Left shift is multiply by 2 and Right shift is divide by 2.

`bnot 0` is `-1` so I guess in some sense an infinite stream of ones?! Arbitrary doesn't mean infinite, so I don't know if there is a lot of room for interpretation 🤔

Reply:
My problem was actually with whether left shift means multiply or divide by 2. I am so picky now, because it's the first time (afaik) the representation of integers has impact on semantics. I know this topic is quite standardized, but since we are on a blockchain, I would rather refer to a particular standard (or the protocol if it's there) or describe more precisely what those operations do.

Examples would help too.